x-pack/plugins/cases/public/components/case_view/metrics/index.tsx

```diff
@@ -7,26 +7,34 @@
 
 import React from 'react';
 import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner, EuiPanel } from '@elastic/eui';
+import { useGetCaseMetrics } from '../../../containers/use_get_case_metrics';
 import { CaseViewMetricItems } from './totals';
-import { CaseViewMetricsProps } from './types';
 import { CaseStatusMetrics } from './status';
+import { useCasesFeatures } from '../../cases_context/use_cases_features';
 
-export const CaseViewMetrics: React.FC<CaseViewMetricsProps> = React.memo(
-  ({ metrics, features, isLoading }) => (
+export const CaseViewMetrics = React.memo(({ caseId }: { caseId: string }) => {
+  const { metricsFeatures } = useCasesFeatures();
+  const { data, isLoading } = useGetCaseMetrics(caseId, metricsFeatures);
+
+  if (!isLoading && metricsFeatures.length === 0) {
```

Comment:
I think we should remove the loading check. Otherwise, the component will render, shows the loading spinner, and then disappear.